### PR TITLE
Checkmarx: Do not hardcode Active and Verified finding attributes

### PR DESCRIPTION
**Pull request by ssiriz**
Issue #4598

### CODE REVIEW

File: dojo/tools/checkmarx/parser.py [hunk 1]
```diff
@@ -119,8 +119,9 @@ def process_result_file_name_aggregated(self, dupes, findingdetail, query, resul
             find = Finding(title=title,
                            cwe=int(cwe),
                            test=self.test,
-                           active=False,
-                           verified=False,
+                           # These lines are commented so the default active/verified will not be set as False (Relate to issue #4598)
```
Comment (StefanFl) [inline dojo/tools/checkmarx/parser.py:122]: All 3 lines with comments should go completely. With the git versioning and the link to issue and pull request there is all the information why the lines have been deleted.

**Review by StefanFl — DISMISSED** [review]

**Comment by StefanFl** [comment]
Thank you for your contribution @ssiriz , the change I have requested should be very easy to solve.

File: dojo/tools/checkmarx/parser.py (hunk 1 above)
Comment (Maffooch) [reply dojo/tools/checkmarx/parser.py:122]: Agreed

**Comment by damiencarol** [comment]
@StefanFl @Maffooch made the changes myself, you could review again

**Comment by Maffooch** [comment]
Checkmarx unit test is totally broke now. Will take a look

Also I rebased this to `dev` and now there is an extra commit. shouldn't be a big deal after squashing. Thoughts?

File: dojo/unittests/tools/test_checkmarx_parser.py [hunk 2]
```diff
@@ -195,10 +195,6 @@ def check_parse_file_with_single_vulnerability_has_single_finding(self, findings
         self.assertEqual("Stored XSS (Users.java)", item.title)
         self.assertEqual(int, type(item.cwe))
         self.assertEqual(79, item.cwe)
-        self.assertEqual(bool, type(item.active))
-        self.assertEqual(False, item.active)
-        self.assertEqual(bool, type(item.verified))
-        self.assertEqual(False, item.verified)
```
Comment (valentijnscholten) [inline dojo/unittests/tools/test_checkmarx_parser.py:201]: would still be good to check if the value is as expected

**Review by valentijnscholten — CHANGES_REQUESTED** [review]
see comment

**Review by Maffooch — APPROVED** [review]
Finally got around to fixing the tests on this one

**Review by valentijnscholten — APPROVED** [review]

**Pull request merged**